packages/react/package.json

```diff
@@ -37,13 +37,12 @@
     "dev": "wait-on ../uploadthing/dist/server.mjs && tsup --watch"
   },
   "dependencies": {
-    "@uploadthing/shared": "^5.2.0",
-    "tailwind-merge": "^1.13.2"
+    "tailwind-merge": "^1.13.2",
+    "uploadthing": "^5.0.0"
```

Comment:
This should be a peerDep still otherwise users might have multiple versions of uploadthing installed in their project

Reply:
When I had it as a peer dep and full dep, the linter yelled about uploadThing being a peer dep but not a dev dep. Should uploadThing just be in peer and dev then?